lib/main.coffee

```diff
@@ -32,3 +44,7 @@ module.exports =
 
     newColumns = undefined if newColumns.length is 0
     atom.config.set('wrap-guide.columns', newColumns)
+
+  toggle: ->
+    currentState = atom.config.get('wrap-guide.enabled')
```

Comment:
That's true, it would make it more annoying, but I think it would also be inconsistent with every other Toggle command in Atom that I can think of (e.g. Toggle Find in Buffer / Project, Toggle Tree View, Toggle Command Palette, Toggle Keybinding Resolver, etc.), none of which save changes to `config.cson`.

I don't feel very strongly about this either way, but I'd lean towards consistency here, I think.